usecases/byteOperations/byteOperations.go

```diff
@@ -0,0 +1,72 @@
+// Package byteOperations provides helper functions to (un-) marshal objects from or into a buffer
+package byteOperations
```

Comment:
seems like we do not use camel case names for files (snake case instead) and dirs (no underscores, hyphens, capitals)

Reply:
renamed